example/extending/newlang/5-hello-python/build.mill

```diff
@@ -0,0 +1,158 @@
+// == Basic Python Pipeline 
+
+// This Example demonstrates integration of https://www.python.org[Python]
+// compilation into a Mill build to compile https://www.python.org[Python] Scripts. Mill
+// does not come bundled with https://www.python.org[Python] integration, so here we begin setting
+// one up from first principles using the Basic `Python3`(Already Present in OS) command Line.
+
+// === Python Setup
+//
+// First, we need to use the `Python3` CLI tool to create virtual environment and
+// install `mypy` for Type Checking using the official Python Virtual Environment
+// https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments[Documentation].
+
+// `MyPy` is a static type checker that uses Python `type hints` to enforce `type correctness` and `catch errors` during development.
+
+package build
+import mill._
+
+def createVenv: T[PathRef] = Task {
+  val venvDir = T.dest / "venv"
+
+  os.call(
+    Seq(
```

Comment:
Don't need the `Seq` Here, don't need the `.toString`, and the `call` should be a one-liner